base/server/python/pki/server/deployment/__init__.py

```diff
@@ -246,6 +246,9 @@ def create_server_xml(self, instance):
         # Configure /etc/pki/<instance>/server.xml
         server_config = instance.get_server_config()
 
+        logger.info('Configuring Tomcat admin port')
+        server_config.set_port(self.mdict['pki_tomcat_server_port'])
+
```

Comment:
This could be done only if not default!

Reply:
Yes, but right now `pki_tomcat_server_port` has a non-empty default value (i.e. `8005`), so it's probably more efficient to just replace it rather than getting the default value from `server.xml` first then comparing it to the user-provided value.

I think later on we can make some additional changes so that `pki_tomcat_server_port` will have an empty default value, then we will update the `server.xml` only if the user overrides the parameter.